opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -166,14 +165,13 @@ public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_wi
     }
 
     public SELF user_denied_consent() {
-        ExtractableResponse<Response> response = RestAssured
+        ExtractableResponse<Response> response = withSignatureHeaders(RestAssured
```

Comment:
This function is called within the user session (from UI), not Fintech - it is protected only by users' cookie and not by request signing - no signature headers here

Reply:
Removed signature header adding for non-fintech requests